### ISSUE TITLE
Facets that don't work

### ISSUE THREAD

**Issue by werg** [error-central/diffenv#16]
Circling back to what we talked about earlier (I forget where?):

We need a robust policy for writing facets or a way to automatically catch if the command a facet calls is not installed.

I'd argue for now we should maybe make it standard to have every shell script just run something like this and return a standard error if not installed:

https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script

What do you think?

**Comment by wanderingstan** [comment]
Ah yes, that's what I suggested in this PR: https://github.com/error-central/diffenv/pull/13

But now I'm leaning toward keeping everything just dumb and simple. Like, what if we just returned everything from the facet (stderr+stdout) as it's value. So then a facet that runs just `composer --version` would have a value of `-bash: composer: command not found`. It would look like:
```
php:
    composer-version: "-bash: composer: command not found"
    composer-show: "-bash: composer: command not found"
```

It's not the prettiest thing in the world, but everyone will know exactly what it means: "this person doesn't have composer installed".  and there is no extra code needed to be smart about determining if the command exists.
